docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/untar.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_label: untar
+description: Extracts the contents of a tar (or gzipped tar) file to a specified location.
+---
+
+# `untar`
+
+`untar` extracts the contents of a tar file (including gzipped tar files) to a specified location. It automatically detects if the file is gzipped and handles it accordingly.
+
+## Configuration
+
+| Name | Type | Required | Description |
+|------|------|----------|-------------|
+| `filePath` | `string` | Y | Path to the tar file to extract. This path is relative to the temporary workspace that Kargo provisions for use by the promotion process. |
```

Comment:
`inPath` feels as if it would be more "symmetric" with `outPath`. The `copy` step has already used `inPath` along with `outPath`. I think making that change here would help keep things more consistent overall.